account_financial_report/reports.xml

```diff
@@ -17,7 +17,7 @@
       model="report_general_ledger"
       string="General Ledger"
       report_type="qweb-html"
-      name="account_financial_report.report_general_ledger_qweb"
+      name="account_financial_report.report_general_ledger_html_qweb"
```

Comment:
This is for this reason [https://github.com/OCA/account-financial-reporting/issues/558](https://github.com/OCA/account-financial-reporting/issues/558)